parser.go

```diff
@@ -2595,6 +2614,107 @@ func (p *Parser) parseCreateVectorIndex(pos token.Pos) *ast.CreateVectorIndex {
 	}
 }
 
+func (p *Parser) parseCreateSearchIndex(pos token.Pos) *ast.CreateSearchIndex {
+	p.expectKeywordLike("SEARCH")
+	p.expectKeywordLike("INDEX")
+	name := p.parseIdent()
+	p.expect("ON")
+	tableName := p.parseIdent()
+	p.expect("(")
+	columnName := parseSeparatedList(p, ",", p.parseIdent)
+	rparen := p.expect(")").Pos
+
+	storing := p.tryParseStoring()
+
+	var partitionColumns []*ast.Ident
+	if p.tryExpect("PARTITION") != nil {
```

Comment:
OK, `tryExpect` and `tryExpectKeywordLike` will be removed.
It may be more acceptable if the method doesn't return a value, like `func (*Parser).tryConsume()`, but it's not very useful.

Reply:
Already removed https://github.com/cloudspannerecosystem/memefish/pull/111/commits/4969ac556a2f07ef27fdb5785fd17e1ae397f122